File: src/SextantTG/SextantTG.WebServices/DTO/TourItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SextantTG.WebServices.DTO
{
    public class TourItem : DTO
    {
        public string TourId { get; set; }
        public string TourName { get; set; }
        public string Status { get; set; }
        public DateTime BeginDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: src/App/src/com/ss/stg/dto/TourItem.java
package com.ss.stg.dto;

import java.util.Date;

public class TourItem extends DTO {

	public TourItem() {
	}

	private String tourId;
	private String tourName;
	private String status;
	private Date beginDate;
	private Date endDate;
	private String userId;

	public String getTourId() {
		return tourId;
	}

	public void setTourId(String tourId) {
		this.tourId = tourId;
	}

	public String getTourName() {
		return tourName;
	}

	public void setTourName(String tourName) {
		this.tourName = tourName;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public Date getBeginDate() {
		return beginDate;
	}

	public void setBeginDate(Date beginDate) {
		this.beginDate = beginDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}
}
